04-kogito-travel-agency/README.md

```diff
@@ -158,6 +158,33 @@ To run the generated native executable, generated in `target/`, execute
 ./target/kogito-travel-agency-{version}-runner
 ```
 
+### Start Kogito Data Index Service
+
+ We have to start the Data Index Service, we can find this service description more detailed at [Kogito Data Index Service](https://github.com/kiegroup/kogito-runtimes/wiki/Data-Index-Service).
```

Comment:
I would actually point people to runner jar from maven - either central for released version or to jboss repo where we have snapshots

Reply:
+1